src/style.css

```
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+form {
+  position: relative;
+}
+
+.input {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: block;
+  margin: 20px 0;
+  width: 100%;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.big {
+  height: 70px;
+  padding: 25px 0 23px 62px;
```

```suggestion
  height: 70px;
  padding-left: 62px;
```
Let's try to set only left padding